examples/step-90/doc/results.dox

```diff
@@ -0,0 +1,26 @@
+<h1>Results</h1>
+
+The numerical solution for a very finest mesh is shown below.
```

Comment:
What is shown? I guess you have created the iso-surface in Paraview?

I think it would make also sense to show 2D results, then you could show the result at the background DoFs. Or alternately, show the results in 3D but with the mesh intersected by plane or so.